index.js

```diff
@@ -600,6 +648,133 @@ herman.annotations = [
   },
 
   /**
+   * Custom `@font` annotation.
+   * For remotely-hosted fonts, expects font name followed by parenthetical list
+   * of variants to display on first line, with optional html head as nested
+   * block.
+   * For locally-hosted fonts, expects font name and bracketed list of formats
+   * on first line, optionally followed by parenthetical list of variants
+   * (defaults to all variants).
+   * NOTE: Locally-hosted fonts require global `fontpath` Herman setting.
+   */
+  function font(env) {
+    const valid_formats = ['ttf', 'otf', 'woff', 'woff2', 'svg', 'svgz', 'eot'];
+    // Matches first occurence of text within `''` or `""`
+    const keyRE = /(["'])(?:(?=(\\?))\2.)*?\1/;
+    // Matches text within `()`
+    const variantsRE = /\(([^)]*)\)/;
+    // Matches text within `{}`
+    const formatsRE = /{([^}]*)}/;
```

Comment:
@davisagli @wlonk Here are the other three REs I've added; happy for feedback.

Reply:
So, there is one thing in these that gives me pause:

`url("http://some/wacky/url/with/a/)/in/it")`

You wanna deal with quoted values differently, you may need a context-free grammar, so I guess it's really "do you even want to deal with that?"

Reply:
Gah, I didn't think about it that URLs could potentially contain parentheses. So, to deal with this correctly, if the URL is not quoted then I just wait for the first `)`, but if it is quoted I could just wait for the next `")` or `')`, yes?

Reply:
Technically no, since quotes can be escaped (https://www.w3.org/TR/CSS2/syndata.html#value-def-uri). Surely there's an existing library for parsing CSS that we could use instead of discovering the edge cases ourselves?

Reply:
@davisagli I can look. The regex I'm using (https://github.com/oddbird/sassdoc-theme-herman/pull/177/files#diff-9f97a9c36fbbfb0d19068d6fae3bb84eR85) takes escaped quotes into account.